src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -213,6 +219,32 @@ async def post_init(self) -> None:
             health_pb2.HealthCheckResponse.SERVING,
         )
 
+    def _make_generator(
+        self,
+        prompt: str,
+        prompt_token_ids: list[int],
+        **kwargs,
+    ):
+        # V1 removed inputs in favor of prompt
```

Comment:
Using `inputs` was [deprecated in the V0 engine client](https://github.com/vllm-project/vllm/blob/v0.7.3/vllm/engine/multiprocessing/client.py#L478) and has been removed in the V1 engine.